### PR TITLE
fix: Fix average entry valuation calculation

### PR DESCRIPTION
**Pull request by Jiajia-Cui**

### CODE REVIEW

File: protocol/0042-LIQF-setting_fees_and_rewarding_lps.md [hunk 1]
```diff
@@ -109,10 +109,33 @@ The equity-like share for each LP is then
 (LP i equity-like share) = (LP i virtual stake) / (sum over j from 1 to N of (LP j virtual stake)).
 ```
 
-The average entry valuation (which should be reported by the APIs) is defined, at the time of change of an LP commitment as:
+The average entry valuation (which should be reported by the APIs) is defined, at the time of change of an LP commitment as folows:
```
Comment (davidsiska-vega) [inline protocol/0042-LIQF-setting_fees_and_rewarding_lps.md]:
```suggestion
The average entry valuation (which should be reported by the APIs and could be calculated only by the data node as it doesn't itself impact core state) is defined, at the time of change of an LP commitment as folows:
```

File: protocol/0042-LIQF-setting_fees_and_rewarding_lps.md [hunk 2]
```diff
@@ -109,10 +109,33 @@ The equity-like share for each LP is then
 (LP i equity-like share) = (LP i virtual stake) / (sum over j from 1 to N of (LP j virtual stake)).
 ```
```
Comment (davidsiska-vega) [inline protocol/0042-LIQF-setting_fees_and_rewarding_lps.md:115]:
```suggestion

There is a [Google sheet - requiring Vega login](https://docs.google.com/spreadsheets/d/14AgZwa6gXVBUFBUUOmB7Y9PsG8D4zmYN/edit#gid=886563806) showing this.

```

File: protocol/0042-LIQF-setting_fees_and_rewarding_lps.md [hunk 3]
```diff
@@ -109,10 +109,33 @@ The equity-like share for each LP is then
...
+```
+Example 3: 
+An existing LP has `average entry valuation 1090.9` and `S=110`. Currently the sum of all virtual stakes is `3000`. They wish to remove `20` from their stake. Their average entry valuation stays the same
+```
+(average entry valuation) = 1090.9
+```
+
 
 
 There is a [Google sheet - requiring Vega login](https://docs.google.com/spreadsheets/d/14AgZwa6gXVBUFBUUOmB7Y9PsG8D4zmYN/edit#gid=886563806) showing this.
```
Comment (davidsiska-vega) [inline protocol/0042-LIQF-setting_fees_and_rewarding_lps.md]:
```suggestion
```

**Review by davidsiska-vega — DISMISSED** [review]
LGTM but I'd like at least on of {@EVODelavega , @cdummett} to approve before merging.

File: protocol/0042-LIQF-setting_fees_and_rewarding_lps.md [hunk 4]
```diff
@@ -109,13 +109,37 @@ The equity-like share for each LP is then
...
+2. Calculate the entry valuation at the time stake `Delta S` is added / removed as 
+```
+(entry valuation) = sum over j from 1 to N of (LP j virtual stake)
+```
+3. Update the average entry valuation to 
+```
+(average entry valuation) <- (average entry valuation) x S / (S + Delta S) + (entry valuation) x (Delta S) / (S + Delta S)
+```
+Example 1: 
+1. A new LP has `0` stake and they wish to add `Delta S = 100`. Currently the sum of all virtual stakes is `1000`. The average entry valuation is
```
Comment (cdummett) [inline protocol/0042-LIQF-setting_fees_and_rewarding_lps.md]: Remove numbering "1."

File: protocol/0042-LIQF-setting_fees_and_rewarding_lps.md [hunk 5]
```diff
@@ -109,13 +109,37 @@ The equity-like share for each LP is then
...
+```
+(average entry valuation) = 1090.9
 ```
 
 
-There is a [Google sheet - requiring Vega login](https://docs.google.com/spreadsheets/d/14AgZwa6gXVBUFBUUOmB7Y9PsG8D4zmYN/edit#gid=886563806) showing this.
+
 
 
 **Check** the sum from over `i` from `1` to `N` of `LP i equity-like share` is equal to `1`.
```
Comment (cdummett) [inline protocol/0042-LIQF-setting_fees_and_rewarding_lps.md]: Move this "Check" line above section defining average-entry-valuation formulas so it is straight after ELS formulas.

**Review by cdummett — COMMENTED** [review]
Small formatting changes for readability. Otherwise formulas and numbers make sense to me.

**Review by cdummett — DISMISSED** [review]
LGTM

File: protocol/0042-LIQF-setting_fees_and_rewarding_lps.md [hunk 6]
```diff
@@ -108,17 +108,37 @@ The equity-like share for each LP is then
...
+(average entry valuation) <- 1000 x 100 / (100 + 10) + 2000 x 10 / (100 + 10) = 1090.9.... 
+```
+Example 3: 
+An existing LP has `average entry valuation 1090.9` and `S=110`. Currently the sum of all virtual stakes is `3000`. They wish to remove `20` from their stake. Their average entry valuation stays the same
+```
+(average entry valuation) = 1090.9
+```
 
-**Check** the sum from over `i` from `1` to `N` of `LP i equity-like share` is equal to `1`.
 **Warning** the above will be decimal calculations so the above checks will only be true up to a rounding errors.
```
Comment (davidsiska-vega) [inline protocol/0042-LIQF-setting_fees_and_rewarding_lps.md]: This should have gone together with the **Check** @Jiajia-Cui

**Review by davidsiska-vega — CHANGES_REQUESTED** [review]

**Review by EVODelavega — DISMISSED** [review]

File: protocol/0042-LIQF-setting_fees_and_rewarding_lps.md [hunk 7]
```diff
@@ -108,17 +108,37 @@ The equity-like share for each LP is then
...
-The average entry valuation (which should be reported by the APIs) is defined, at the time of change of an LP commitment as:
-```
-(Avg entry valuation i) = sum over j from 1 to N of (LP j virtual stake)
-```
-
+**Check** the sum from over `i` from `1` to `N` of `LP i equity-like share` is equal to `1`.
 
 There is a [Google sheet - requiring Vega login](https://docs.google.com/spreadsheets/d/14AgZwa6gXVBUFBUUOmB7Y9PsG8D4zmYN/edit#gid=886563806) showing this.
 
+The average entry valuation (which should be reported by the APIs and could be calculated only by the data node as it doesn't itself impact core state) is defined, at the time of change of an LP commitment as folows:
```
Comment (candida-d) [inline protocol/0042-LIQF-setting_fees_and_rewarding_lps.md]:
```suggestion
The average entry valuation (which should be reported by the APIs and could be calculated only by the data node as it doesn't itself impact core state) is defined, at the time of change of an LP commitment as follows:
```

**Review by davidsiska-vega — APPROVED** [review]

**Pull request merged**